cirq-core/cirq/transformers/routing/route_circuit_cqc_test.py

```diff
@@ -107,6 +107,41 @@ def test_circuit_with_measurement_gates():
     cirq.testing.assert_same_circuits(routed_circuit, circuit)
 
 
+def test_circuit_with_intermediate_multi_qubit_measurement_gates():
```

Comment:
please simplify the test. you can break it into two tests one for a valid circuit and one invalid